binding-mruby/binding-util.cpp

```diff
@@ -72,7 +72,7 @@ static const MrbExcData excData[] =
 	{ PHYSFS,   "PHYSFSError" },
 	{ SDL,      "SDLError"    },
 	{ MKXP,     "MKXPError"   },
-	{ IO,       "IOError"     }
+	{ IO,       "IOError2"     }
```

Comment:
Why the rename? Is `IOError` already defined by mruby?

Reply:
Yeah. If I remember right, this throws up some error about `IOError` already existing.

Reply:
Should I rename this to something better?

Reply:
This is from the io-gem too, right. Just delete this entry in the array, and query the existing error class like done here: https://github.com/Ancurio/mkxp/blob/master/binding-mruby/binding-util.cpp#L120

Reply:
Did this, but tbh I've no idea what's going on here